components/service/firefox-accounts/src/main/java/mozilla/components/service/fxa/manager/SyncEnginesStorage.kt

```diff
@@ -46,5 +46,12 @@ class SyncEnginesStorage(private val context: Context) {
         storage().edit().putBoolean(engine.nativeName, status).apply()
     }
 
+    /**
+     * Clears out any stored [SyncEngine] state.
+     */
+    internal fun clear() {
+        storage().edit().clear().apply()
```

Comment:
Could we add a test for this ? :)

Reply:
There's a test in FxaAccountManager, but maybe an explicit one wouldn't hurt as well :)